### ISSUE TITLE
Feature Request?: Email Notification Upon Reply to Commentor

### ISSUE THREAD

**Issue by ERTurner** [dokufreaks/plugin-discussion#221]
I'm not sure if this is a feature that isn't currently available with this plugin, or an issue with my installation, but I expected that when "reply"ing to a comment, an email would go out to the person who posted the comment that I replied to, letting them know that someone has replied.

Person A posts a comment
Person B replies to that comment
Person A gets an email that a reply has been posted to their comment

This is different than subscribing to the whole discussion, or moderators receiving notifications when comments are posted.

Any chance of getting this functionality in the future?

**Comment by maddin18** [comment]
Indeed, to me it would be very useful if the pluging might be configured to create notifications on _any_ change, no matter if it is a new thread, a response or an edit.

The use case behind this is:
Besides the original wiki idea with rather static content, my users would like to communicate via the same platform (dokuwiki). This is similar to a messenger. However, the content is integrated in the wiki page and there is no more need to agree on a certain messenger (signal, threema, WA, ...). Sounds crazy, but my users are doing this today. What they are missing most is are e-mail notifications as described above.

Is this feasible? Would a tip help to make it happen? Is there a simple code change which I might apply myself?

**Comment by Klap-in** [comment]
Notification on all change (page edits+all comments) is already available, by using DokuWiki's subscription system.
If that is enabled (enable the 'subscribers' configuration setting), then the Discussion plugin uses that to notify on any comment as well. Because DokuWiki subscription page have to be used, the subscription checkbox below the comment form is hidden for the logged-in users. 
See also https://www.dokuwiki.org/subscription

There is not a notification option only for replies. Pull request welcome.
